taxonomies/ingredients_processing.txt

```diff
@@ -222,12 +222,12 @@ de:ungeschält, ungeschälte
 
 #en:description:To remove the outer covering or shell of something
 en:shelled
-es:sin cascara
+es:sin cascara, pelada, pelado, peladas, pelados
 fr:décortiqué, décortiquée, décortiqués, décortiquées, décoquillé, décoquillée, décoquillés, décoquillées
 
 # en:description:With the outermost layer or skin removed
 en:peeled
-es:sin piel
+es:sin piel, repelada, repelado, repeladas, repelados, descascarrilado, descascarrilada, descascarrilados, descascarriladas
```

Comment:
Be careful not to empty a dictionary here. It should really appear on in ingredients. We want to limit the processing power needed.

Reply:
I thought the idea was split ingredients from processing? These are all processes. Unfortunately with spanish there are 2 genders and singular/plural so pretty much everything has 4 possible states. I have examples of these processes (at least a single occurance of each) which is why I entered it.